pkg/reconciler/broker/broker.go

```diff
@@ -296,11 +296,11 @@ func (r *Reconciler) reconcileUsingCRD(ctx context.Context, b *eventingv1.Broker
 
 	if b.Spec.Delivery != nil && b.Spec.Delivery.DeadLetterSink != nil {
 		queue, err := r.rabbit.ReconcileQueue(ctx, &triggerresources.QueueArgs{
-			Name:        naming.CreateBrokerDeadLetterQueueName(b),
-			Namespace:   b.Namespace,
-			Owner:       *kmeta.NewControllerRef(b),
-			Labels:      triggerresources.QueueLabels(b, nil),
-			ClusterName: b.Spec.Config.Name,
+			Name:      naming.CreateBrokerDeadLetterQueueName(b),
+			Namespace: b.Namespace,
+			Broker:    b,
```

Comment:
```suggestion
			ClusterName:    b.Spec.Config.Name,
			ClusterNamespace:    b.Spec.Config.Namepace,
```

I was hoping to keep these APIs at a different level of abstraction by not including objects like the Broker and the Trigger. Like they should only know "RabbitMQ"-y things.

Reply:
OK, I will pass in `ClusterName` & `ClusterNamespace` instead of `Broker`.